### DISCUSSION TITLE
Сustom fields when uploading images

### DISCUSSION THREAD

**Discussion by Pitcentr** [directus/directus#13130] [Archive]
Hello, tell me how to fill in custom fields when uploading images to the directus_files collection, thank you

![изображение](https://user-images.githubusercontent.com/34168062/166891101-a7a9854d-30cb-443c-8aaf-52ea646776a8.png)

**Comment by rijkvanzanten** [comment]
You can send any of the other fields on the file as parts in the multipart request when uploading the file. See https://docs.directus.io/reference/files/#upload-a-file for an example 🙂
